pkg/reduce/pnf/metrics.go

```diff
@@ -60,8 +60,8 @@ var dropBytesCount = promauto.NewCounterVec(prometheus.CounterOpts{
 
 // platformError is used to indicate the number of Internal/Platform errors
 var platformError = promauto.NewCounterVec(prometheus.CounterOpts{
-	Subsystem: "platform",
```

Comment:
To be consistent with and also distinguished from map forwarder's platform error metric.

Reply:
do we need to change the grafana dashboard?